index.js

```diff
@@ -96,7 +96,7 @@ function registerListener(session, options, callback = () => {}) {
 			}
 
 			if (options.showBadge && ['darwin', 'linux'].includes(process.platform)) {
-				app.badgeCount = activeDownloadItems();
+				app.setBadgeCount(activeDownloadItems());
```

Comment:
You are moving back to the soft deprecated methods. I would prefer to only use the property.